src/pages/p2p/_availability.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react'
+import styled from 'styled-components'
+import Desktop from 'images/svg/p2p-desktop.svg'
+import Mobile from 'images/svg/p2p-mobile.svg'
+import { localize, Localize } from 'components/localization'
+import { Header, LocalizedLinkText, Text } from 'components/elements'
+import { Flex, SectionContainer } from 'components/containers'
+import Login from 'common/login'
+import device from 'themes/device'
+
+const Row = styled.div`
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+`
+
+const Section = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    padding: 8rem 12rem;
+    max-width: 1600px;
+    margin: 0 auto;
+    align-items: center;
+    justify-content: center;
+    box-shadow: inset 0 1px 0 0 #f2f3f4;
```

Comment:
you can use --color-grey-8 css variables for this

Reply:
Alright, will be incorporated